packages/cashier/src/components/cashier-locked/__tests__/cashier-locked.spec.tsx

```diff
@@ -1,51 +1,42 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
 import CashierLocked from '../cashier-locked';
-import { useDepositLocked } from '@deriv/hooks';
-import { TRootStore } from 'Types';
+import { useCashierLocked, useDepositLocked } from '@deriv/hooks';
+import { mockStore } from '@deriv/stores';
+import { TRootStore } from '../../../types';
 import CashierProviders from '../../../cashier-providers';
 
 jest.mock('@deriv/hooks', () => ({
     ...jest.requireActual('@deriv/hooks'),
     useDepositLocked: jest.fn(() => false),
+    useCashierLocked: jest.fn(() => false),
 }));
-
-jest.mock('@deriv/hooks', () => ({
-    ...jest.requireActual('@deriv/hooks'),
-    useDepositLocked: jest.fn(() => false),
-}));
-
-jest.mock('@deriv/hooks', () => ({
-    ...jest.requireActual('@deriv/hooks'),
-    useDepositLocked: jest.fn(() => false),
-}));
+const mockUseDepositLocked = useDepositLocked as jest.MockedFunction<typeof useDepositLocked>;
+const mockUseCashierLocked = useCashierLocked as jest.MockedFunction<typeof useCashierLocked>;
 
 describe('<CashierLocked />', () => {
+    beforeEach(() => {
+        mockUseDepositLocked.mockReturnValue(false);
```

Comment:
We need mocking these before each test case bc if we mock them once in one test, they're gonna remain mocked with that value for the rest of the tests of a suite